cms/templates/index.html

```diff
@@ -485,6 +500,54 @@ <h3 class="title">${_('Create Your First Library')}</h3>
       </div>
       %endif
 
+      % if is_programs_enabled:
+        % if len(programs) > 0:
+          <div class="programs programs-tab">
+            <!-- Classes related to courses are intentionally reused here, to duplicate the styling used for course listing. -->
+            <ul class="list-courses">
+              % for program in sorted(programs, key=lambda p: p['name'].lower()):
```

Comment:
is there a reason to manipulate the ordering here?  seems like it would be better to push that logic down into the server API call.